src/Tests/Service/RegistrationServiceTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupU2fBundle\Tests\Service;
+
+use Mockery as m;
+use PHPUnit_Framework_TestCase as TestCase;
+use Surfnet\StepupU2fBundle\Service\RegistrationService;
+use Surfnet\StepupU2fBundle\Service\RegistrationVerificationResult;
+use u2flib_server\Error;
+
+final class RegistrationServiceTest extends TestCase
+{
+    const APP_ID = 'https://gateway.surfconext.nl/u2f/app-id';
```

Comment:
please use a url that may never exist (i.e. replace `.nl` with `.invalid` or something like that)